### PR TITLE
CRN-848 Validation error to be displayed on the URL field when user does not specify the domain

### PR DESCRIPTION
**Pull request by niezgoda**
Expected Behaviour: User should be directed to the URL field with an error message ‘Please enter a valid URL’. This should  not be displayed at the top of the page.

### CODE REVIEW

File: packages/react-components/src/templates/ContactInfoModal.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ const ContactInfoModal: React.FC<ContactInfoModalProps> = ({
             <LabeledTextField
               title="Website 1"
               subtitle="(Optional)"
-              pattern={USER_SOCIAL_WEBSITE.source}
+              pattern={UrlExpression}
```
Comment (markcsims) [inline packages/react-components/src/templates/ContactInfoModal.tsx:116]: Should this be tested?

File: packages/react-components/src/templates/ContactInfoModal.tsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ const ContactInfoModal: React.FC<ContactInfoModalProps> = ({
             <LabeledTextField
               title="Website 2"
               subtitle="(Optional)"
-              pattern={USER_SOCIAL_WEBSITE.source}
```
Comment (markcsims) [inline packages/react-components/src/templates/ContactInfoModal.tsx:129]: Should this be tested?

File: apps/crn-server/test/controllers/research-outputs.test.ts [hunk 3]
```diff
@@ -1038,7 +1044,9 @@ describe('ResearchOutputs controller', () => {
         expect(squidexGraphqlClientMock.request).toHaveBeenCalledWith(
           expect.anything(),
           expect.objectContaining({
-            filter: `(data/link/iv eq '${researchOutputRequest.link}')`,
+            filter: `(data/link/iv eq '${sanitiseForSquidex(
+              researchOutputRequest.link || '',
+            )}')`,
```
Comment (markcsims) [inline apps/crn-server/test/controllers/research-outputs.test.ts:1049]: I don't think we should be asserting using sanitiseForSquidex. It's from the production code, and it's part of the method under test.

**Review by markcsims — CHANGES_REQUESTED** [review]

File: packages/react-components/src/templates/ContactInfoModal.tsx (hunk 2 above)
Comment (niezgoda) [reply packages/react-components/src/templates/ContactInfoModal.tsx:129]: image

**Review by markcsims — APPROVED** [review]

**Pull request merged**